DependencyInjection/Configuration.php

```diff
@@ -76,6 +76,11 @@ public function getConfigTreeBuilder()
                     ->info("The user interface language localization to use.")
                     ->example("en-au")
                 ->end()
+                ->scalarNode('disable_native_spell_checker')
+                    ->defaultNull()
+                    ->info("Disabled the browser's spell checker.")
```

Comment:
Please change this to match the Ckeditor documentation: "Disables the built-in spell checker if the browser provides one."